docs/howto/object-storage/s3/public-bucket.md

```diff
@@ -0,0 +1,129 @@
+---
+# Public buckets
+
+{{page.meta.description}}
+Making a bucket globally readable entails setting a *bucket policy* that enables read access on all its objects.
+
+> Setting a bucket policy affects **all** objects in a bucket.
+> To avoid inadvertent disclosure of existing information, consider setting public read policies only on empty buckets.
+
+## Prerequisites
+
+Object versioning requires that you configure your environment with [working S3-compatible credentials](credentials.md).
+
+## Setting a public read policy for a bucket
+
+First, create a local policy file named `policy.json`, with the following content (replace `<bucket-name>` with the name of your bucket):
+
+```json
+{ 
+  "Statement": [ 
+    {
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::<bucket-name>/*"
+    }
+  ]
+}
+```
+
+To apply this policy to a bucket such that read-only access is permitted for everyone, run the following command:
+
+=== "aws"
+    ```bash
+    aws --profile <region> \
+      --endpoint-url=https://s3-<region>.{{brand_domain}}:8080 \
+      s3api put-bucket-policy \
+      --policy file://policy.json \
+      --bucket <bucket-name>
+    ```
+=== "mc"
+    ```bash
+    mc anonymous set-json policy.json <region>/<bucket-name>
+    ```
+=== "s3cmd"
+    ```bash
+    s3cmd -c ~/.s3cfg-<region> setpolicy policy.json s3://<bucket-name>
+    ```
+
+## Accessing objects in a public bucket
+
+To access an object in a public bucket from a web browser or a generic HTTP/HTTPS client like `curl`, you must construct its URI as follows:
+
+```plain
+https://s3-<region>.{{brand_domain}}:8080/<project-uuid>:<bucket-name>/object-name
+```
+
+> Your project UUID is listed as the `project_id` field in the output of the `openstack ec2 credentials create` command you used to [create your S3-compatible credentials](credentials.md).
+>
+> If you did not note it down at the time of account creation, you can always retrieve it with `openstack ec2 credentials <access-key>`.
+
+For example, to retrieve an object named `bar.pdf` in a bucket named `foo` that belonged to the project with the UUID `07576783684248f7b2745e34356c6025` in the {{brand}} Kna1 region, you would run:
+
+```console
+$ curl -O https://s3-kna1.{{brand_domain}}:8080/07576783684248f7b2745e34356c6025:foo/bar.pdf
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100 62703  100 62703    0     0   186k      0 --:--:-- --:--:-- --:--:--  186k
+```
+
+## Enabling bucket listing
+
+The `policy.json` file above allows anyone to retrieve known objects by name, but does not enable listing the bucket's contents.
+Most of the time, this is what you want.
+
+However, in the event that you do need unauthenticated clients to be able to list all objects in a bucket, modify your `policy.json` file as in the following example:
```

Comment:
I would use a simpler syntax ("in the event" reminds me of the instructions of your friendly flight attendant): "However, if you do need unauthenticated clients".

Reply:
I went with "in case" rather than "in the event that". That's more concise, but it still implies that it should be the exception, rather than the norm (whereas a simple "if" strikes me as completely neutral).

Reply:
Ah, yes, "in case" is more suitable!